Twitter_API/code.py

```diff
@@ -71,7 +71,7 @@ def get_bearer_token():
                     caption_position=(5, 210),
                     caption_color=0x808080)
 
-bearer_token = get_bearer_token()
+bearer_token = secrets['twitter_bearer_token']
```

Comment:
@dglaude  This works and we don't need to generate the token anymore - hurray! Could you please remove the `get_bearer_token` and `halt_and_catch_fire` functions since they are unnecessary now and not called elsewhere?